frontend/src/cards/DisparityBarChartCard.tsx

```diff
@@ -79,6 +80,19 @@ function DisparityBarChartCardWithKey(props: DisparityBarChartCardProps) {
               row[props.breakdownVar] !== UNKNOWN_RACE
           );
 
+        let shouldShowDoesntAddUpMessage = false;
+        if (
+          props.breakdownVar === "race_and_ethnicity" &&
```

Comment:
Can we import this string from metric config just in case it ever changes in the future?

Reply:
good call